test-helpers/src/main/java/com/okta/testhelpers/NetworkDispatcher.kt

```diff
@@ -60,13 +60,9 @@ class NetworkDispatcher : Dispatcher() {
     }
 
     override fun dispatch(request: RecordedRequest): MockResponse {
-        var matchedEntry: Entry? = null
         val oktaRequest = OktaRecordedRequest(request)
-        enqueuedResponses.forEach { entry ->
-            if (entry.requestMatcher(oktaRequest)) {
-                matchedEntry = entry
-                return@forEach
-            }
```

Comment:
This didn't seem to return the first matched request, but instead the last matched request. return@forEach didn't seem to stop forEach. So, I replaced .forEach with .first